dlrover/python/elastic_agent/torch/training.py

```diff
@@ -110,6 +110,7 @@ class ElasticLaunchConfig(LaunchConfig):
     """
 
     network_check: bool = False
+    quick_check: bool = False
```

Comment:
`check_round_num: int = 2` maybe better. We can set `check_round_num=1` if we want to quickly check the node health.